MAUI/NavigationDrawer/Navigation-Pane-Sides.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: post
+title: Configuring The Drawer In .NET MAUI Navigation Drawer | Syncfusion
+description: Learn here all about Configuring The Drawer In Different Sides support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+# Configuring The Drawer In Different Sides in .NET MAUI Navigation Drawer
+
+The supplemental pane can be drawn in and out from all four sides. [`Position`]() property is used to change the side of the pane and the values are
+
+*	[`Left`]()
+
+*	[`Right`]()
+
+*	[`Top`]()
+
+*	[`Bottom`]()
+
+N> The default position is [`Left`]().
+
+## Left
+
+The navigation pane draws in and out from the [`Left`]() side. It can be set as shown below:
```

Comment:
The navigation pane pulled in and out from the `Left` side. It can be set as shown below:

Reply:
Changed